zipline/pipeline/factors/statistical.py

```diff
@@ -148,11 +155,6 @@ class RollingLinearRegression(CustomFactor, SingleInputMixin):
         The factor/slice whose columns are the predictor/independent variable
         of each regression with `dependent`. If `independent` is a Factor,
         regressions are computed asset-wise.
-    independent : zipline.pipeline.Term with a numeric dtype
```

Comment:
This is an unrelated change to the rest of the PR.  We had this documented twice.